app/models/manageiq/providers/redhat/infra_manager/event_catcher/runner.rb

```diff
@@ -41,7 +41,8 @@ def monitor_events
 
   def queue_event(event)
     _log.info "#{log_prefix} Caught event [#{event[:name]}]"
-    EmsEvent.add_queue('add_rhevm', @cfg[:ems_id], event.to_hash)
+    event_hash = ManageIQ::Providers::Redhat::InfraManager::EventParser.event_to_hash(event, @cfg[:ems_id])
+    EmsEvent.add_queue('add', @cfg[:ems_id], event_hash)
```

Comment:
absolutely. good spot. Looks like this a xml thing
https://github.com/ManageIQ/ovirt/blob/master/lib/ovirt/service.rb#L155-L168
@gauravaradhye or could you open a PR to fix this?

@cben how did this PR affect your (I assume containers?) provider?